app/post.go

```diff
@@ -798,20 +799,32 @@ func (a *App) parseAndFetchChannelIdByNameFromInFilter(channelName, userId, team
 }
 
 func (a *App) searchPostsInTeam(teamId string, userId string, paramsList []*model.SearchParams, modifierFun func(*model.SearchParams)) (*model.PostList, *model.AppError) {
-	channels := []store.StoreChannel{}
+	wg := new(sync.WaitGroup)
```

Comment:
Why not use `var wg sync.WaitGroup` instead?

Reply:
I used short variable declaration style inside the function to follow the current convention. I can change it back to normal declaration if it's more explicit.

Reply:
Ok, the equivalent short variable declaration should be something like wg := sync.WaitGroup{}. When you use `new` you generating a `*sync.WaitGroup` not a `sync.WaitGroup`, which is not a problem in this case, I just wanted to know the reasoning behind. We use short variable declaration normally, but is not weird to see variables that are used with "zero value" with just a declaration. We are not using a lot `new` in our code, so... I would say, change it to the declaration approach, which is the one used in the example documentation for the wait groups